### PR TITLE
PDCT-164 : Implement 'CPR Document Status' on CCLW CSV

### PR DESCRIPTION
**Pull request by diversemix**
# Description

Only adds the column to the expected CSV schema

Contributes to https://linear.app/climate-policy-radar/issue/PDCT-164/ensure-documents-families-marked-with-status-archived-are-removed

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] The PR represents a single feature (small driveby fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: tests/core/ingestion/test_family.py [hunk 1]
```diff
@@ -160,6 +160,31 @@ def test_family_document_from_row__updates(test_db: Session):
...
+
+    assert list(result.keys()) == ["family_document"]
+
+    db_family_doc = (
+        test_db.query(FamilyDocument)
+        .filter(FamilyDocument.import_id == DOCUMENT_IMPORT_ID)
+        .one()
+    )
+    assert db_family_doc == family_document
+    assert db_family_doc.document_status == "DELETED"
```
Comment (THOR300) [inline tests/core/ingestion/test_family.py:185]: I thought our document status's were CREATED, PUBLISHED, ARCHIVED so are we just setting the status to whatever is defined in the row.document_status?

From reading the update_if_enum_changed function it looks like we would be setting any defined status?

**Review by THOR300 — COMMENTED** [review]
Just one comment regarding whether we are defining and enforcing a set of distinct document status's.

File: tests/core/ingestion/test_family.py (hunk 1 above)
Comment (THOR300) [reply tests/core/ingestion/test_family.py:185]: Discussed offline, the document status' are validated via an enum on the document status column of the family object in the database. 

The options are also now DELETED, PUBLISHED, CREATED

**Review by THOR300 — APPROVED** [review]
LGTM post chat

**Review by joel-wright — APPROVED** [review]
lgtm

**Pull request merged**